### DISCUSSION TITLE
Few news curves for you

### DISCUSSION THREAD

**Discussion by ghost** [bbc-mc/sdweb-merge-block-weighted-gui#41] [Show and tell]
`
EXPONENTIAL_A	0.018,0.026,0.038,0.055,0.078,0.113,0.162,0.234,0.336,0.483,0.695,1.000,0.000,0.018,0.026,0.038,0.055,0.078,0.113,0.162,0.234,0.336,0.483,0.695,1.000
EXPONENTIAL_B	1.000,0.695,0.483,0.336,0.234,0.162,0.113,0.078,0.055,0.038,0.026,0.018,0.000,1.000,0.695,0.483,0.336,0.234,0.162,0.113,0.078,0.055,0.038,0.026,0.018
EXPONENTIAL_C	1.000,0.695,0.483,0.336,0.234,0.162,0.113,0.078,0.055,0.038,0.026,0.018,0.000,0.018,0.026,0.038,0.055,0.078,0.113,0.162,0.234,0.336,0.483,0.695,1.000
EXPONENTIAL_D	0.018,0.026,0.038,0.055,0.078,0.113,0.162,0.234,0.336,0.483,0.695,1.000,0.000,1.000,0.695,0.483,0.336,0.234,0.162,0.113,0.078,0.055,0.038,0.026,0.018
FIBONACCI_A	0.000,0.011,0.011,0.022,0.034,0.056,0.090,0.146,0.236,0.382,0.618,1.000,0.000,0.000,0.011,0.011,0.022,0.034,0.056,0.090,0.146,0.236,0.382,0.618,1.000
FIBONACCI_B	1.000,0.618,0.382,0.236,0.146,0.090,0.056,0.034,0.022,0.011,0.011,0.000,0.000,1.000,0.618,0.382,0.236,0.146,0.090,0.056,0.034,0.022,0.011,0.011,0.000
FIBONACCI_C	1.000,0.618,0.382,0.236,0.146,0.090,0.056,0.034,0.022,0.011,0.011,0.000,0.000,0.000,0.011,0.011,0.022,0.034,0.056,0.090,0.146,0.236,0.382,0.618,1.000
FIBONACCI_D	0.000,0.011,0.011,0.022,0.034,0.056,0.090,0.146,0.236,0.382,0.618,1.000,0.000,1.000,0.618,0.382,0.236,0.146,0.090,0.056,0.034,0.022,0.011,0.011,0.000
LOGARITHMIC_A	0.000,0.087,0.167,0.241,0.310,0.375,0.435,0.492,0.547,0.598,0.647,0.693,0.000,0.000,0.087,0.167,0.241,0.310,0.375,0.435,0.492,0.547,0.598,0.647,0.693
LOGARITHMIC_B	1.000,0.913,0.833,0.759,0.690,0.625,0.565,0.508,0.453,0.402,0.353,0.307,0.000,1.000,0.913,0.833,0.759,0.690,0.625,0.565,0.508,0.453,0.402,0.353,0.307
LOGARITHMIC_C	1.000,0.913,0.833,0.759,0.690,0.625,0.565,0.508,0.453,0.402,0.353,0.307,0.000,0.307,0.353,0.402,0.453,0.508,0.565,0.625,0.690,0.759,0.833,0.913,1.000
LOGARITHMIC_D	0.307,0.353,0.402,0.453,0.508,0.565,0.625,0.690,0.759,0.833,0.913,1.000,0.000,1.000,0.913,0.833,0.759,0.690,0.625,0.565,0.508,0.453,0.402,0.353,0.307
SIGMOID_A	0.007,0.016,0.040,0.093,0.204,0.388,0.612,0.796,0.907,0.960,0.984,0.993,0.000,0.007,0.016,0.040,0.093,0.204,0.388,0.612,0.796,0.907,0.960,0.984,0.993
SIGMOID_B	0.993,0.984,0.960,0.907,0.796,0.612,0.388,0.204,0.093,0.040,0.016,0.007,0.000,0.993,0.984,0.960,0.907,0.796,0.612,0.388,0.204,0.093,0.040,0.016,0.007
SIGMOID_C	0.993,0.984,0.960,0.907,0.796,0.612,0.388,0.204,0.093,0.040,0.016,0.007,0.000,0.007,0.016,0.040,0.093,0.204,0.388,0.612,0.796,0.907,0.960,0.984,0.993
SIGMOID_D	0.007,0.016,0.040,0.093,0.204,0.388,0.612,0.796,0.907,0.960,0.984,0.993,0.000,0.993,0.984,0.960,0.907,0.796,0.612,0.388,0.204,0.093,0.040,0.016,0.007
QUADRATIC_A	0.000,0.008,0.033,0.074,0.132,0.207,0.298,0.405,0.529,0.669,0.826,1.000,0.000,0.000,0.008,0.033,0.074,0.132,0.207,0.298,0.405,0.529,0.669,0.826,1.000
QUADRATIC_B	1.000,0.992,0.967,0.926,0.868,0.793,0.702,0.595,0.471,0.331,0.174,0.000,0.000,1.000,0.992,0.967,0.926,0.868,0.793,0.702,0.595,0.471,0.331,0.174,0.000
QUADRATIC_C	1.000,0.992,0.967,0.926,0.868,0.793,0.702,0.595,0.471,0.331,0.174,0.000,0.000,0.000,0.174,0.331,0.471,0.595,0.702,0.793,0.868,0.926,0.967,0.992,1.000
QUADRATIC_D	0.000,0.174,0.331,0.471,0.595,0.702,0.793,0.868,0.926,0.967,0.992,1.000,0.000,1.000,0.992,0.967,0.926,0.868,0.793,0.702,0.595,0.471,0.331,0.174,0.000
SINUS_A	0.000,0.142,0.282,0.415,0.541,0.655,0.756,0.841,0.910,0.959,0.990,1.000,0.000,0.000,0.142,0.282,0.415,0.541,0.655,0.756,0.841,0.910,0.959,0.990,1.000
SINUS_B	1.000,0.858,0.718,0.585,0.459,0.345,0.244,0.159,0.090,0.041,0.010,0.000,0.000,1.000,0.858,0.718,0.585,0.459,0.345,0.244,0.159,0.090,0.041,0.010,0.000
SINUS_C	1.000,0.858,0.718,0.585,0.459,0.345,0.244,0.159,0.090,0.041,0.010,0.000,0.000,0.000,0.010,0.041,0.090,0.159,0.244,0.345,0.459,0.585,0.718,0.858,1.000
SINUS_D	0.000,0.010,0.041,0.090,0.159,0.244,0.345,0.459,0.585,0.718,0.858,1.000,0.000,1.000,0.858,0.718,0.585,0.459,0.345,0.244,0.159,0.090,0.041,0.010,0.000
ROOT_SQUARE_A	0.000,0.302,0.426,0.522,0.603,0.674,0.739,0.798,0.853,0.905,0.953,1.000,0.000,0.000,0.302,0.426,0.522,0.603,0.674,0.739,0.798,0.853,0.905,0.953,1.000
ROOT_SQUARE_B	1.000,0.698,0.574,0.478,0.397,0.326,0.261,0.202,0.147,0.095,0.047,0.000,0.000,1.000,0.698,0.574,0.478,0.397,0.326,0.261,0.202,0.147,0.095,0.047,0.000
ROOT_SQUARE_C	1.000,0.698,0.574,0.478,0.397,0.326,0.261,0.202,0.147,0.095,0.047,0.000,0.000,0.000,0.047,0.095,0.147,0.202,0.261,0.326,0.397,0.478,0.574,0.698,1.000
ROOT_SQUARE_D	0.000,0.047,0.095,0.147,0.202,0.261,0.326,0.397,0.478,0.574,0.698,1.000,0.000,1.000,0.698,0.574,0.478,0.397,0.326,0.261,0.202,0.147,0.095,0.047,0.000
EXPONENTIAL_INV_A	0.000,0.365,0.597,0.744,0.838,0.897,0.935,0.958,0.974,0.983,0.989,0.993,0.000,0.000,0.365,0.597,0.744,0.838,0.897,0.935,0.958,0.974,0.983,0.989,0.993
EXPONENTIAL_INV_B	0.000,0.365,0.597,0.744,0.838,0.897,0.935,0.958,0.974,0.983,0.989,0.993,0.000,1.000,0.635,0.403,0.256,0.162,0.103,0.065,0.042,0.026,0.017,0.011,0.007
EXPONENTIAL_INV_C	1.000,0.635,0.403,0.256,0.162,0.103,0.065,0.042,0.026,0.017,0.011,0.007,0.000,0.007,0.011,0.017,0.026,0.042,0.065,0.103,0.162,0.256,0.403,0.635,1.000
EXPONENTIAL_INV_D	0.007,0.011,0.017,0.026,0.042,0.065,0.103,0.162,0.256,0.403,0.635,1.000,0.000,1.000,0.635,0.403,0.256,0.162,0.103,0.065,0.042,0.026,0.017,0.011,0.007
HYPERBOLIC_A	0.000,0.083,0.154,0.214,0.267,0.312,0.353,0.389,0.421,0.450,0.476,0.500,0.000,0.000,0.083,0.154,0.214,0.267,0.312,0.353,0.389,0.421,0.450,0.476,0.500
HYPERBOLIC_B	1.000,0.917,0.846,0.786,0.733,0.688,0.647,0.611,0.579,0.550,0.524,0.500,0.000,1.000,0.917,0.846,0.786,0.733,0.688,0.647,0.611,0.579,0.550,0.524,0.500
HYPERBOLIC_C	1.000,0.917,0.846,0.786,0.733,0.688,0.647,0.611,0.579,0.550,0.524,0.500,0.000,0.500,0.524,0.550,0.579,0.611,0.647,0.688,0.733,0.786,0.846,0.917,1.000
HYPERBOLIC_D	0.500,0.524,0.550,0.579,0.611,0.647,0.688,0.733,0.786,0.846,0.917,1.000,0.000,1.000,0.917,0.846,0.786,0.733,0.688,0.647,0.611,0.579,0.550,0.524,0.500

`

**Comment by Luke2642** [comment]
![image](https://github.com/user-attachments/assets/f8f89a57-2b5a-4f8b-a7ff-fa107f1cb034)

**Comment by Luke2642** [comment]
Huge discontinuities can lead to bad results if the checkpoints are too different, but the asymmetry can be good too. Smoothstep*3 and /3 and reversed are actually quite interesting.

It would be good if there were sliders for the entire in block and entire out block, so you could do a simple, 0.3 on in blocks 0.6 on out blocks, or vice versa.

If you constructed them as lego brick blobs, you could have a 'blob adder':
"position of blob", "width of blob," "height of blob", "symmetry of blob", "sharpness of blobs" 
[0 to 25], [0 to 25], [0 to 1], [-1 to 1], [0-1] [add blob to values below] [replace values below]

all the presets could be defined as blobs, or created using multiple blobs, and the user clicks the buttons updates the apply the blob to the sliders below.
